apps/vs-agent/src/testVtrRoutes.ts

```diff
@@ -32,61 +32,78 @@ import {
 } from '@credo-ts/core'
 import Ajv from 'ajv/dist/2020'
 import addFormats from 'ajv-formats'
+import axios from 'axios'
 import { createHash } from 'crypto'
 import express from 'express'
 import * as fs from 'fs'
 import * as path from 'path'
 
+import {
+  AGENT_INVITATION_IMAGE_URL,
+  AGENT_LABEL,
+  TESTVTR_ORG_ADDRESS,
+  TESTVTR_ORG_COUNTRYCODE,
+  TESTVTR_ORG_REGISTRYID,
+  TESTVTR_ORG_REGISTRYURL,
+  TESTVTR_ORG_TYPE,
+  TESTVTR_SERVICE_DESCRIPTION,
+  TESTVTR_SERVICE_MINIMUMAGEREQUIRED,
+  TESTVTR_SERVICE_PRIVACYPOLICY,
+  TESTVTR_SERVICE_TERMSANDCONDITIONS,
+  TESTVTR_SERVICE_TYPE,
+} from './config'
 import { VsAgent } from './utils/VsAgent'
 
 // Load schemas from data.json at startup (used for schema validation and mock responses)
 const ecsSchemas = JSON.parse(fs.readFileSync(path.resolve(process.cwd(), 'data.json'), 'utf-8'))
 const ajv = new Ajv({ strict: false })
 addFormats(ajv)
+const allowedLogTags = ['ecs-service', 'ecs-org'] as const
+type AllowedLogTag = (typeof allowedLogTags)[number]
 
 // Main function to add all test routes to the Express app
-export const addSelfVtrRoutes = async (app: express.Express, agent: VsAgent, publicApiBaseUrl: string) => {
+export const addTestVtrRoutes = async (app: express.Express, agent: VsAgent, publicApiBaseUrl: string) => {
   // Create a Verifiable Presentation for ECS Service
   // Verifiable JsonSchemaCredential
   // Register endpoints for example Verifiable Presentations
   await registerVerifiablePresentationEndpoint(
-    '/self-vtr/ecs-service-c-vp.json',
+    '/test-vtr/ecs-service-c-vp.json',
```

Comment:
```suggestion
    '/self-vt/ecs-service-c-vp.json',
```
We have objects belonging to our "self trust registry" and "self verifiable public registry". Maybe it is enough to englobe all of them into a "self verifiable trust" namespace?

Or should we use 'self-tr' for JSON schema credentials and 'self-vpr' for the root JSON Schemas?